example/package.json

```diff
@@ -2,19 +2,20 @@
   "private": true,
   "scripts": {
     "dev": "cross-env DEBUG=vite-ssg:* vite",
-    "build": "cross-env DEBUG=vite-ssg:* vite-ssg build"
+    "build": "vite-ssg build"
```

Comment:
Why this gets removed tho? We are not using it now but possibility will do in the future.

Reply:
Oh, because I have a lot of env that dotenv will print it out. Can you put it back? Thanks.

Reply:
Not very sure what you are talking about, what will print out?

Reply:
dotenv(npm package) will print my user/xxx/dotfiles/.env file which is not related to this repo, I don't know why.😅